lib/talis-cdk-stack/talis-region.ts

```diff
@@ -9,3 +9,20 @@ export enum TalisShortRegion {
   EU = "eu",
   LOCAL = "local",
 }
+
+const talisRegionToShortRegionMap = new Map<TalisRegion, TalisShortRegion>();
+talisRegionToShortRegionMap.set(TalisRegion.CANADA, TalisShortRegion.CANADA);
+talisRegionToShortRegionMap.set(TalisRegion.EU, TalisShortRegion.EU);
+talisRegionToShortRegionMap.set(TalisRegion.LOCAL, TalisShortRegion.LOCAL);
+
+export function getTalisShortRegionFromTalisRegionName(
+  talisRegionName: string
+): TalisShortRegion | undefined {
+  const talisRegion = Object.values(TalisRegion).find(
+    (talisRegion) => talisRegion === talisRegionName
+  );
+  if (!talisRegion) {
+    return undefined;
+  }
+  return talisRegionToShortRegionMap.get(talisRegion);
+}
```

Comment:
While your approach is valid, it's a bit verbose.
In JavaScript/TypeScript we could write it shorter:

```suggestion
const talisRegionToShortRegionMap = {
  [TalisRegion.CANADA]: TalisShortRegion.CANADA,
  [TalisRegion.EU]: TalisShortRegion.EU,
  [TalisRegion.LOCAL]: TalisShortRegion.LOCAL,
};

export function getTalisShortRegionFromTalisRegion(
  talisRegion: TalisRegion
): TalisShortRegion | undefined {
  return talisRegionToShortRegionMap[talisRegion] ?? undefined;
}
```

There's nothing wrong with using Maps over plain Objects, but Maps are mostly useful when they are filled dynamically and/or when keys are not primitive values. For static mapping, object literal syntax is easier to use and read.